### ISSUE TITLE
JsonSerialize method not compatible

### ISSUE THREAD

**Issue by ideal1sm** [Benjacho/belongs-to-many-field-nova#122]
Hello, today I installed the package and got an error 

![image](https://user-images.githubusercontent.com/42361491/226904142-9498bd80-5793-49c2-9db0-3ef1ffb91d27.png)

> Declaration of Benjacho\BelongsToManyField\BelongsToManyField::jsonSerialize() must be compatible with Laravel\Nova\Fields\Field::jsonSerialize(): array

I solved this problem by adding to the jsonSerialize method of the BelongsToManyField class, return type of array

`public function jsonSerialize(): array`

**Comment by DzonyBB** [comment]
Yes, the error is gone, but the problem is not solved. Now there is no error, but it does not display any data. Help me set it up, thank you.
